packages/manager/modules/vps/src/vps-action.service.js

```diff
@@ -1,12 +1,13 @@
-class VpsActionService {
+export default class VpsActionService {
+  /* @ngInject */
   constructor(CucControllerHelper) {
     this.CucControllerHelper = CucControllerHelper;
   }
 
   rescue(serviceName) {
     return this.CucControllerHelper.modal.showModal({
       modalConfig: {
-        templateUrl: 'app/vps/modal/shortcut/password/vps-password.html',
+        templateUrl: 'vps/modal/shortcut/password/vps-password.html',
```

Comment:
Couldn't be turned into a regular import statement?

Reply:
As it works before. We should refactor the modals in VPS in the future to use states and components (and use regular import to use templates).